test/test_train_mp_imagenet_fsdp.py

```diff
@@ -222,7 +222,7 @@ def train_imagenet():
   # - you may wrap sub-modules at different granularity (e.g. at each resnet
   #   stage or each residual block or each conv layer).
   fsdp_wrap = lambda m: FSDP(
-      m.to(device),
+      m,
```

Comment:
Perhaps a newbie question, I wonder how the code in the PR tested.

Reply:
Hi @vanbasten23, I've run the MNIST + ImageNet FSDP test cases as commented above, and I'm still running more tests.